guides/code_review_process.md

```diff
@@ -3,16 +3,17 @@
 Git workflows play a crucial role in the code review process. Two notable methodologies are *GitHub Flow* and *GitFlow*. Despite their similar names, they represent distinct concepts. GitHub Flow is characterized by a modern and simplified approach, contrasting with Gitflow's comprehensive branching model.
 
 Working with [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow):
+
 1. Create an issue, if not existing yet, to discuss what you want to implement
-2. Fork the repository and create your branch from master
-3. Push changes to your fork and open a pull request 
+2. Fork the repository and create your branch from main
```

Comment:
@cheld I guess `main` is more common nowadays

Reply:
Good catch.

Which lint tool have you been using? I already was thinking to implement a pipeline for markdown changes 
that is doing some linting and checking the hyper links